build.yaml

```diff
@@ -1139,6 +1139,216 @@ steps:
      - batch_pods_ns
      - hail_run_image
      - build_hail
+ - kind: runImage
+   name: test_hail_python_local_backend_0
+   image:
+     valueFrom: hail_run_image.image
+   resources:
+     memory: "7.5G"
+     cpu: "2"
+   script: |
+     set -ex
+     cd /io
+     tar xzf test.tar.gz
+     tar xzf resources.tar.gz
+     tar xzf data.tar.gz
+     tar xvf wheel-container.tar
+     python3 -m pip install --no-dependencies hail-*-py3-none-any.whl
+     export HAIL_TEST_RESOURCES_DIR=./resources
+     export HAIL_DOCTEST_DATA_DIR=./data
+     export HAIL_TEST_BUCKET=hail-test-dmk9z
+     export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
+     export PYSPARK_SUBMIT_ARGS="--driver-memory 6g pyspark-shell"
```

Comment:
I'm guessing some of these aren't needed in local mode. Is it worth deleting them to keep things tidy?

Reply:
oh, I can delete the pyspark one